editoast/src/cors.rs

```diff
@@ -0,0 +1,25 @@
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::{Request, Response};
+
+/// Attack this fairing to add CORS headers to responses of your routes.
+pub struct Cors;
+
+#[rocket::async_trait]
+impl Fairing for Cors {
+    fn info(&self) -> Info {
+        Info {
+            name: "CORS",
+            kind: Kind::Response,
+        }
+    }
+
+    async fn on_response<'r>(&self, _request: &'r Request<'_>, response: &mut Response<'r>) {
+        response.set_header(Header::new(
+            "Access-Control-Allow-Methods",
+            "PATCH, DELETE, HEAD, GET, POST, PUT, OPTIONS",
+        ));
+        response.set_header(Header::new("Access-Control-Allow-Headers", "*"));
+        response.set_header(Header::new("Access-Control-Allow-Origin", "*"));
```

Comment:
that's not great, can't we have an option to restrict CORS a bit? We can leave it as is for now, but I'd very much like to have a TODO-list of web app hardening things to do